schema/admins/area.yaml

```diff
@@ -0,0 +1,26 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Area Schema
+description: >-
+  Area adds land or maritime polygon to locality.
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    required: [localityId]
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+    properties:
+      localityId:
+        description: References specific feature of locality type
+        type: string
+      maritime:

```

Comment:
`isMaritime` to reflect that it is a boolean property?

Reply:
+1
I'd also vote that this property be refactored out to an `admins/defs.yaml` file and reused in administrativeBoundary to be consistent.

Reply:
I wanted to do that but administrativeBoundary already had `maritime`, hence used same, but I agree, will replace in both types to `isMaritime`

Reply:
Done.